Sim/SimDelphesInterface/src/DelphesSimulation.h

```diff
@@ -70,45 +77,66 @@ class DelphesSimulation: public GaudiAlgorithm {
   // Finalize.
   virtual StatusCode finalize();
 
-  // Convert internal Delphes objects to FCC EDM objects - UPDATE
-  void  ConvertParticle(TObjArray *  Input, fcc::ParticleCollection * coll );
-  void  ConvertJet(     TObjArray *  Input, fcc::GenJetCollection   * coll );
-  void  ConvertMET(     TObjArray *  Input, fcc::ParticleCollection * coll );
-  void  ConvertHT(      TObjArray *  Input, fcc::ParticleCollection * coll );
-
 private:
 
   // Delphes detector card to be read in
-  std::string              m_DelphesCard;
-
-  // Output collections to be write out
-  std::vector<std::string> m_outCollections;
+  std::string            m_DelphesCard;
 
   // Delphes
   Delphes               *m_Delphes ;
   DelphesFactory        *m_DelphesFactory ;
   DelphesExtHepMCReader *m_HepMCReader ;
```

Comment:
Nah. The point was to have the delphes interface not reading the HepMC file directly, but getting the HepMC event from the FCC framework; in that case the Delphes internal HepMCReader is not needed, right?